parmed/formats/pdb.py

```diff
@@ -722,7 +722,7 @@ def print_atoms(atom, coords):
                 else:
                     atoms = sorted(res.atoms, key=lambda atom: atom.number)
                 for atom in atoms:
-                    pa, others, (x, y, z) = print_atoms(atom, coords[0])
+                    pa, others, (x, y, z) = print_atoms(atom, coord)
```

Comment:
`print_atoms` is an internal function defined and used within a single scope.  When I came up with the name `print_atoms`, `print` was an adjective to me (it returns the printed atom), not a verb.  Its name matters little.